models/user_model.py

```diff
@@ -93,6 +94,10 @@ def delete_user(username):
         db.session.commit()
         return done
 
+    def update_last_login(self):
+        self.last_login = datetime.datetime.utcnow()
```

Comment:
Consider using a decorator or base method to handle db.session.commit() consistently across model changes